src/univariate/continuous/normal.jl

```diff
@@ -74,22 +73,175 @@ kurtosis(d::Normal{T}) where {T<:Real} = zero(T)
 
 entropy(d::Normal) = (log2π + 1)/2 + log(d.σ)
 
-
 #### Evaluation
+gradlogpdf(d::Normal, x::Real) = (d.μ - x) / d.σ^2
+function logpdf(d::Normal, x::Real)
+    if iszero(d.σ)
+        d.μ == x ? Inf : -Inf
+    else
+        -(((x - d.μ) / d.σ)^2 + log2π) / 2 - log(d.σ)
+    end
+end
+function pdf(d::Normal, x::Real)
+    if iszero(d.σ)
+        d.μ == x ? Inf : 0.0
+    else
+        exp(-((x - d.μ) / d.σ)^2 / 2) * invsqrt2π / d.σ
+    end
+end
+function cdf(d::Normal, x::Real)
+    if iszero(d.σ)
+        float(d.μ ≤ x)
+    else
+        erfc(-(x - d.μ) / d.σ * invsqrt2) / 2
+    end
+end
+function logcdf(d::Normal, x::Real)
+    if iszero(d.σ)
+        d.μ ≤ x ? 0.0 : -Inf
+    else
+        z = (x - d.μ) / d.σ
+        z < -1.0 ? log(erfcx(-z * invsqrt2)/2) - abs2(z)/2 : log1p(-erfc(z * invsqrt2)/2)
+    end
+end
+
+function xval(μ::Real, σ::Real, z::Number)
+    if isinf(z) && iszero(σ)
+        μ + one(σ) * z
+    else
+        μ + σ * z
+    end
+end
 
-@_delegate_statsfuns Normal norm μ σ
+function quantile(d::Normal, p::Real)
```

Comment:
and a `cquantile` (for values < `eps()`).

Reply:
Added `cquantile`.